src/components/Organisms/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useRef, useState, useEffect } from "react";
+import styled from "styled-components";
+import FONTFAMILY from "../../variables/font_family";
+import TEXTS from "../../variables/texts";
+import COLOR from "../../variables/color";
+import AddTaskButton from "../Atoms/AddTaskButton/index";
+import Tasks from "../Molecules/index";
+
+const TodoCard = () => {
+  const [taskList, setTaskList] = useState([{ name: "", initializing: true }]);
+
+  const onAddTaskButtonClick = () => {
+    const addList = [...taskList];
+    setTaskList(addList.concat({ name: "", initializing: true }));
```

Comment:
```suggestion
    setTaskList([...taskList, { name: "", initializing: true }]);
```
別のオブジェクトとして管理するのはGoodです！
このようにすると1行でかけたりします。

Reply:
deleteList, changeList も同様